experimental/dds/tree/src/Common.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import { ITelemetryBaseEvent, ITelemetryProperties } from '@fluidframework/common-definitions';
```

Comment:
```suggestion
import { ITelemetryBaseEvent } from '@fluidframework/common-definitions';
```
I think your build's about to fail with a lint error...

Reply:
removed